### PR TITLE
Redid the guide completely

### PR DESCRIPTION
**Pull request by fix-ON**

### CODE REVIEW

File: Multiple/Dual Frequency Oscillator/gerbers/BOM.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+## BOM
+
+Here's a BOM (bill of materials) of all the parts you need for the different DFO boards. 
+Go to mouser.com and type in the part. no. Due to the pandemic and part shortage, many parts needed are still out of stock. ;(
```
Comment (tevey) [inline Multiple/Dual Frequency Oscillator/gerbers/BOM.md:4]: I would suggest too remove the second part of this line since it will be outdated.
In general try to avoid time-sensitive information since it will end up outdated.

File: Multiple/Dual Frequency Oscillator/gerbers/BOM.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+| 1x | 	    1.8V LDO		        | 595-TPS78218DDCT
+| 1x | 	    27 MHz crystal	        | 717-9B-27.000MAAJ-B
+| 3x | 	    1 uF capacitor	        | 81-GRM188R61E105KA12
+| 1x | 	    2.2 uF capacitor	        | 81-GRM188R61E225KA2D
+| 1x | 	    18 Ohm resistor	        | 71-CRCW0603-18-E3
+| 1x | 	    10 kOhm resistor	        | 71-CRCW0603-10K-E3
+| 1x | 	    22 kOhm resistor	        | 71-CRCW0603-22K-E3
+| 1x | 	    Precision pin header		 | [AW 122/20 (Reichelt)](http://www.reichelt.de/Sockets-IC/AW-122-20/3/index.html?ACTION=3&GROUPID=3215&ARTICLE=4426&OFFSET=500&WKID=0&)
+
+DFO DFO 5V SMD:
```
Comment (tevey) [inline Multiple/Dual Frequency Oscillator/gerbers/BOM.md:22]: "DFO" word twice

File: Multiple/Dual Frequency Oscillator/gerbers/BOM.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+| 1x | 	3.3V LDO		        | 595-TPS78233DDCR
+| 1x | 	1.8V LDO		        | 595-TPS78218DDCT
+| 1x | 	27 MHz crystal	        | 717-9B-27.000MAAJ-B
+| 3x | 	1 uF capacitor	        | 81-GRM188R61E105KA12
+| 1x | 	2.2 uF capacitor	        | 81-GRM188R61E225KA2D
+| 1x | 	18 Ohm resistor	        | 71-CRCW0603-18-E3
+| 1x | 	10 kOhm resistor	        | 71-CRCW0603-10K-E3
+| 1x | 	22 kOhm resistor	        | 71-CRCW0603-22K-E3
+
+DFO DFO 3.3V SMD:
```
Comment (tevey) [inline Multiple/Dual Frequency Oscillator/gerbers/BOM.md:37]: "DFO" word twice

File: Multiple/Dual Frequency Oscillator/README.md [hunk 4]
```diff
@@ -1,41 +1,74 @@
...
+
+## DFO PCB's
+
+There are three types of DFO's. Each one are suited for different consoles:
+
+* DFO 5V DIL14 - Mega Drive, Amiga
+* DFO 5V SMD - Playstation, NEO GEO MV-1C
+* DFO 3.3V SMD - SNES
+
+When you know what DFO you need, program it with the correct HEX-files (timing files). More on this later.
```
Comment (tevey) [inline Multiple/Dual Frequency Oscillator/README.md:16]: Consider removing or moving this line since it has nothing to do with the PCB.

File: Multiple/Dual Frequency Oscillator/README.md [hunk 5]
```diff
@@ -1,41 +1,74 @@
...
+
+There are three types of DFO's. Each one are suited for different consoles:
+
+* DFO 5V DIL14 - Mega Drive, Amiga
+* DFO 5V SMD - Playstation, NEO GEO MV-1C
+* DFO 3.3V SMD - SNES
+
+When you know what DFO you need, program it with the correct HEX-files (timing files). More on this later. 
+
+Pre-programmed DFO's are hard to find in stock these days, so you should consider making your own DFO PCB's. Hence the reason for this guide.
```
Comment (tevey) [inline Multiple/Dual Frequency Oscillator/README.md:18]: Maybe reword the last part of the PCB section something like this:
```markdown
If you don't already have a PCB or can't manage to buy one, you can order them from [JLCPCB](https://jlcpcb.com/) or [PCBWay](https://pcbway.com/) by uploading the gerber files located [here](gerber/)
```

File: Multiple/Dual Frequency Oscillator/README.md [hunk 6]
```diff
@@ -1,41 +1,74 @@
...
+When you know what DFO you need, program it with the correct HEX-files (timing files). More on this later. 
+
+Pre-programmed DFO's are hard to find in stock these days, so you should consider making your own DFO PCB's. Hence the reason for this guide. 
+
+If you're familiar with [JLCPCB](https://jlcpcb.com/) or [PCBWay](https://pcbway.com/), download the DFO gerber files [here](https://github.com/modda-se/wiki/tree/main/Multiple/Dual%20Frequency%20Oscillator/images).
+If not, do some research and come back. 
+
+## Configuration
+
+Before you start programming, you should have a Raspberry Pi running Raspbian or similar. Either connected through SSH (recommended) or directly, we need to make sure `ARM I2C interface` is enabled, by typing:
```
Comment (tevey) [inline Multiple/Dual Frequency Oscillator/README.md:25]: Maybe change "directly" to 
```markdown
Either connected through SSH (recommended) or typing directly on the RPi
```

File: Multiple/Dual Frequency Oscillator/README.md [hunk 7]
```diff
@@ -1,41 +1,74 @@
...
 
 Type the following:
 
-
        cd Downloads
        git clone https://github.com/fix-ON/cdceprog_DFO.git
-       cd cdceprog_DFO
-
-
+       cd cdceprog
```
Comment (tevey) [inline Multiple/Dual Frequency Oscillator/README.md:71]: won't the name of the cloned folder be cdceprog_DFO?

File: Multiple/Dual Frequency Oscillator/README.md [hunk 8]
```diff
@@ -60,8 +93,8 @@ Connect the DFO pins to the corresponding points on your specific motherboard ve
 - [Installation guide for Playstation](https://www.consolesunleashed.com/guides/sony-playstation-dual-frequency-oscillator-install-guide/)
 - [Installation guide for Mega Drive](https://www.consolesunleashed.com/guides/sony-playstation-dual-frequency-oscillator-install-guide/)
 
-Playstation SCPH-100X PU-8:
+Playstation Playstation SCPH-100X PU-8:
```
Comment (tevey) [inline Multiple/Dual Frequency Oscillator/README.md:96]: Playstation twice

File: Multiple/Dual Frequency Oscillator/README.md [hunk 9]
```diff
@@ -60,8 +93,8 @@ Connect the DFO pins to the corresponding points on your specific motherboard ve
 - [Installation guide for Playstation](https://www.consolesunleashed.com/guides/sony-playstation-dual-frequency-oscillator-install-guide/)
```
Comment (tevey) [inline Multiple/Dual Frequency Oscillator/README.md:93]: maybe change "Installation guide for Playstation" to "Programming & Installation guide for Playstation"

File: Multiple/Dual Frequency Oscillator/README.md [hunk 10]
```diff
@@ -60,8 +93,8 @@ Connect the DFO pins to the corresponding points on your specific motherboard ve
...
 - [Installation guide for Mega Drive](https://www.consolesunleashed.com/guides/sony-playstation-dual-frequency-oscillator-install-guide/)
```
Comment (tevey) [inline Multiple/Dual Frequency Oscillator/README.md:94]: maybe change "Installation guide for Mega Drive" to "Programming & Installation guide for Mega Drive"

File: Multiple/Dual Frequency Oscillator/README.md [hunk 11]
```diff
@@ -60,8 +93,8 @@ Connect the DFO pins to the corresponding points on your specific motherboard ve
...
 
-Playstation SCPH-100X PU-8:
+Playstation Playstation SCPH-100X PU-8:
 ![Playstation SCPH-100X PU-8 version](images/playstation-dfo-scph-100x-pu-8-3.5V.jpg)  
 
 Example of installed DFO in a Playstation.
-![example](images/example.jpg)
+![example](images/example.JPG)
```
Comment (tevey) [inline Multiple/Dual Frequency Oscillator/README.md:100]: should by small .jpg

**Review by tevey — CHANGES_REQUESTED** [review]
remove README.pdf

File: Multiple/Dual Frequency Oscillator/README.md (hunk 9 above)
Comment (fix-ON) [reply Multiple/Dual Frequency Oscillator/README.md:93]: The website doesn't tell you how to program the DFO.

File: Multiple/Dual Frequency Oscillator/README.md (hunk 10 above)
Comment (fix-ON) [reply Multiple/Dual Frequency Oscillator/README.md:94]: The website doesn't tell you how to program the DFO.

File: Multiple/Dual Frequency Oscillator/gerbers/BOM.md (hunk 1 above)
Comment (fix-ON) [reply Multiple/Dual Frequency Oscillator/gerbers/BOM.md:4]: Good point. Link added to https://octopart.com/

**Review by fix-ON — COMMENTED** [review]
Doing another pull request soon.
